frontend/src/fixtures/commonsPlusFixtures.js

```diff
@@ -11,7 +11,9 @@ const commonsPlusFixtures = {
                 "endingDate": null,
                 "degradationRate": 0.01,
                 "showLeaderboard": false,
+                "capacityPerUser": 50,
                 "carryingCapacity": 100,
+                "effectiveCapacity": 100,
```

Comment:
Please check all of the effectiveCapacities to make sure they are internally consistent.